qml/Launcher/LauncherPanel.qml

```diff
@@ -97,7 +98,7 @@ Rectangle {
                 sourceSize.width: width
                 sourceSize.height: height
                 anchors.centerIn: parent
-                source: "graphics/home.svg"
+                source: root.showCloseIcon ? "graphics/back.svg": "graphics/home.svg"
```

Comment:
Could we make this an `Icon` instead of `Image`? We could drop the `sourceSize` settings here, as that's already handled by `Icon`, and use `name: root.showCloseIcon ? "back" : ""` along with `source: root.showCloseIcon ? "" : "graphics/home.svg"` and avoid having to keep a copy of the icon in unity8 itself, I think. And maybe set `color` property on it, so the back button also gets the right color when it's on.

Reply:
```
Icon {
    objectName: "dashItem"
    width: parent.width * .6
    height: width
    anchors.centerIn: parent
    name: root.showCloseIcon ? "back" : ""
    source: root.showCloseIcon ? "" : "graphics/home.svg"
    color: white
    rotation: root.rotation
}
```

Seems to work for the Ubuntu icon, but it doesn't load Back correctly. Something I did wrong here?

Reply:
No. I misremembered how the source vs name usage there works. :-/

Reply:
So... that idea won't work then?

Reply:
I'm not sure if something slightly more complex might work or not, without a couple simple changes to `Icon` source. Maybe just include the full path to the SVG in `/usr/share/icons/` instead of shipping a copy, for now?

Reply:
That is, don't use `name` property at all, but just the full path in the back case of the ternary statement. If you want to make it a little prettier, you can have something like:

```source: {
    if (root.showCloseIcon) {
        return "/usr/share/icons/.../back.svg";
    } else {
        return "graphics/home.svg"; 
    }
}```